onnxruntime/python/tools/transformers/bert_test_data.py

```diff
@@ -235,6 +235,10 @@ def parse_arguments():
                         default=128,
                         help="maximum sequence length of input")
 
+    parser.add_argument('--input_ids', required=False, type=str, default=None, help="input name for input ids")
```

Comment:
Change the naming to input_ids_name? This argument seems to expect a input name, not the actual input_ids. The naming is confusing. And same for segment_ids and input mask.

Reply:
I changed naming.